PokerGame/PokerForm.cs

```diff
@@ -0,0 +1,206 @@
+
+            switch (i)
+            {
+                case 1:
+                    pictureBox1.Image = image.Value;
+                    pictureBox1.Name = image.Key;
+                    break;
+                case 2:
+                    pictureBox2.Image = image.Value;
+                    pictureBox2.Name = image.Key;
+                    break;
+                case 3:
+                    pictureBox3.Image = image.Value;
+                    pictureBox3.Name = image.Key;
+                    break;
+                case 4:
+                    pictureBox4.Image = image.Value;
+                    pictureBox4.Name = image.Key;
+                    break;
+                case 5:
+                    pictureBox5.Image = image.Value;
+                    pictureBox5.Name = image.Key;
+                    break;
+                default:
+                    break;
+            }
+        }
+
+        private void PlayButton_Click(object sender, EventArgs e)
+        {
+            if(timerShow.Enabled == false)
+            {
+                timerShow.Enabled = true;
+                PlayButton.Enabled = false;
+                poker.BetValue = (int)betNumericUpDown.Value;
+                moneyTextBox.Text = poker.BetValue.ToString();
+                for (int i = 0; i < 5; i++)
+                {
+                    buttons[i].Enabled = false;
+                    if (buttons[i].Text == "Shuffle")
+                    {
+                        poker.GetItIn(pictureBoxes[i].Name, pictureBoxes[i].Image);
+                    } 
+                }
+            }
+        }
+
+        private void timerShow_Tick(object sender, EventArgs e)
+        {
+            if (tickCounter == 5)
+            {
+                tickCounter = 0;
+                timerShow.Enabled = false;
+                int winnings = poker.CalculateWinnings(pictureBoxes[0].Name, pictureBoxes[1].Name, pictureBoxes[2].Name, pictureBoxes[3].Name, pictureBoxes[4].Name);
+                winningsTextBox.Text = winnings.ToString();
+                poker.RenewDeck();
+                timerBetweenHands.Enabled = true;
+
+            }
+            else
+            {
+                tickCounter++;
+                for (int i = 0; i < 5; i++)
+                {
+                    if (buttons[i].Text == "Shuffle")
+                    {
+                        UpdatePokerImages(i + 1);
+                        if (tickCounter == 4)
+                        {
+                            poker.GetItOut(pictureBoxes[i].Name);
+                        }
+                    }
+                }
+            }
+        }
+
+        private void button1_Click(object sender, EventArgs e)
+        {
+            if (button1.Text == "Keep")
+            {
+                button1.Text = "Shuffle";
+            }
+            else if (button1.Text == "Shuffle")
+            {
+                button1.Text = "Keep";
+            }
+        }
```

Comment:
you can make it in one line:
button1.Text = button1.Text == "Keep" ? "Shuffle" : "Keep";